internal/cloud/realm/user.go

```diff
@@ -26,13 +26,49 @@ const (
 // UserState is a Realm application user state
 type UserState string
 
+// String returns the user state string
+func (us UserState) String() string { return string(us) }
+
+// Type returns the user state type
+func (us UserState) Type() string { return "string" }
```

Comment:
this is failing the evergreen linter.  It wants me to make "string" a constant.  Should I do that or suppress the warning? Where would I put the constant, or how do i suppress the warning?